tools/run-all-tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/env bash
+
+set -euo pipefail
+
+script_dir="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)/"
+stdb_root="$(realpath "$script_dir/../")"
+
+# Find cargo.
+cargo=$(which cargo)
+if [ ! -x "$cargo" ] ; then
+    echo "Could not find 'cargo' executable, please install it."
+    exit 1
```

Comment:
will we ever get here? won't `which` return nonzero, causing the script to exit?

Reply:
...you are correct